app/api/user.py

```
@@ -0,0 +1,28 @@
+        natural_text = service.profile_to_text(profile)
+        embedding = service.generate_user_embedding(profile)
+
+        return {
+            "status": "success",
+            "profile": profile.dict(),
+            "natural_text": natural_text,
+            "embedding_shape": embedding.shape,
+            "embedding_sample": embedding[:5].tolist()  
+        }
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"임베딩 생성 중 오류 발생: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)